armi/reactor/blocks.py

```diff
@@ -1197,6 +1200,10 @@ def plotFlux(core, fName=None, bList=None, peak=False, adjoint=False, bList2=Non
         bList2 :
             a separate list of blocks that will also be plotted on a separate axis on the same plot.
             This is useful for comparing flux in some blocks with flux in some other blocks.
+
+        Notes
+        -----
+        This is not a great method. It should be cleand up and migrated into ``utils.plotting``.
```

Comment:
Perhaps a bit self-deprecating, haha. At some point, we should add a note at the top of `blocks.py` providing some of the context behind the method migrations. Beyond the scope of this changeset, though.